src/components/HomePage/HomePage.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Link } from 'react-router-dom';
+import { Fab } from '@material-ui/core/';
+import AddIcon from '@material-ui/icons/Add';
+import Wishlist from '../Wishlist/Wishlist';
+import { handleDeleteWishlist } from '../../client/client';
+
+const useStyles = makeStyles((theme) => ({
+  homeContainer: {
+    height: '100vh',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  controlContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-end',
+  },
+  floatingActionButton: {
+    margin: theme.spacing(1),
+  },
+  wishlistContainer: {
+    width: '50vw',
+  },
+}));
+
+const HomePage = ({ handleGetWishlists }) => {
+  const classes = useStyles();
+  const [wishlists, setWishlists] = useState([]);
+
+  useEffect(() => {
```

Comment:
Excellent use of React Hooks here!  

Though Experiment - useEffect doesn't behave well when passed an async function, but it IS possible to use async/await with useEffect...  What would this useEffect need to look like if it made use of async/await?